xtask/src/install_git_hooks.rs

```diff
@@ -0,0 +1,24 @@
+use clap::Args;
+
+use std::path::Path;
+
+#[derive(Args, Debug)]
+pub struct InstallGitHooks {
+    /// Provide verbose diagnostic output.
+    #[clap(short, long)]
+    verbose: bool,
+}
+
+pub fn install_git_hooks(InstallGitHooks { verbose }: InstallGitHooks) -> cross::Result<()> {
+    let metadata = cross::cargo_metadata_with_args(
+        Some(Path::new(env!("CARGO_MANIFEST_DIR"))),
+        None,
+        verbose,
+    )?
+    .ok_or_else(|| eyre::eyre!("could not find cross workspace and its current version"))?;
```

Comment:
this error message is wrong
```suggestion
    .ok_or_else(|| eyre::eyre!("could not find cross workspace"))?;
```